_data/doc_versions.yml

```diff
@@ -1,5 +1,7 @@
 # Auto-generated by ./scripts/build_doc_version_index.sh
 # DO NOT MODIFY
+- v2.5
```

Comment:
I think that we should not commit this.  Only the new documentations.

Reply:
will be removed